Ex6.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+os.getcwd()
+os.chdir("Intro_Biocom_ND_319_Tutorial6/")
+
+import pandas
+import numpy
+
+game = pandas.read_table("UWvMSU_1-22-13.txt", sep="\t")
+
+import matplotlib.pyplot as plt
+
+WiscScore = game.loc[game['team'] == 'UW', ['score']]
+MSUscore = game.loc[game['team'] == 'MSU', ['score', 'time']]
+
+plt.plot(WiscScore.time, WiscScore,'r-', MSUscore.time,MSUscore,'g-')
+
+### alternatively ###
+
+import plotnine as p9
+
+x = p9.ggplot(game, aes(x='time', y='score', color = 'factor(team)'))
+x+geom_line()
+
```

Comment:
The question is asking to plot the accumulative score of two teams. You want to add up the score for each team from the beginning of the game